app/controllers/games_controller.rb

```diff
@@ -9,7 +9,14 @@ def update
     @game.update_attributes(game_params)
     redirect_to_root_path
   end
-
+  
+  def destroy
+    @game = Game.find(params[:id])
+      return not_found(:forbidden) if @game.user != current_user
+    @game.destroy
+    redirect_to_root_path
```

Comment:
I didn't catch this before: you need a space between `redirect_to` and `root_path`